test/integration/targets/setup_docker/tasks/main.yml

```diff
@@ -30,8 +35,52 @@
       ignore_errors: yes
 
     - set_fact:
+        docker_cli_version: "{{ (docker_cli_version_stdout.stdout | default('0.0')) or '0.0' }}"
         docker_api_version: "{{ docker_api_version_stdout.stdout or '0.0' }}"
         docker_py_version: "{{ docker_py_version_stdout.stdout or '0.0' }}"
 
     - debug:
-        msg: "Docker API version: {{ docker_api_version }}; docker-py library version: {{ docker_py_version }}"
+        msg: "Docker CLI version: {{ docker_cli_version }}; Docker API version: {{ docker_api_version }}; docker-py library version: {{ docker_py_version }}"
+
+    - block:
+      # Cleanup docker daemon
+      - name: "Remove all ansible-test-* docker containers"
+        shell: 'docker ps --no-trunc --format {% raw %}"{{.Names}}"{% endraw %} | grep "^ansible-test-" | xargs -r docker rm -f'
+        register: docker_containers
+      - name: "Remove all ansible-test-* docker volumes"
+        shell: 'docker volume ls --format {% raw %}"{{.Name}}"{% endraw %} | grep "^ansible-test-" | xargs -r docker volume rm -f'
+        register: docker_volumes
+      - name: "Remove all ansible-test-* docker networks"
+        shell: 'docker network ls --no-trunc --format {% raw %}"{{.Name}}"{% endraw %} | grep "^ansible-test-" | xargs -r docker network rm'
+        register: docker_networks
+      - name: Cleaned docker resources
+        debug: var=docker_resources
```

Comment:
It seems like the standard in these tests are using the yaml notation and not `key=value`, would be good to keep the same standard.

Reply:
Fixed.